src/ref.builds

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="*\ref\**\*.*proj">
+    <Project Include="*\ref\**\*.builds;*\ref\**\*.*proj">
```

Comment:
It won't actually build twice, MSBuild is smart enough to prevent that.  If this becomes common-place we'll switch to builds everywhere, but right now that's unnecessary overhead.